framework/doc/content/sqa/framework_srs.md

```diff
@@ -131,7 +137,9 @@ satisfactory resolution.
 !template item key=information-management
 !! information-management-begin
 The core framework in its entirety will be made publicly available on an appropriate repository
-hosting site. Backups and security services will be provided by the hosting service.
+hosting site. Day-to-day backups and security services will be provided by the hosting service. More
+information about MOOSE backups of the public repository on [!ac](INL)-hosted services can be found
+on the following page: [sqa/github_backup.md]
```

Comment:
Should we move the `github_backup.md` file into framework/doc/content? Just worried about apps not including it in their config file.

Reply:
The full path for this file is `framework/doc/content/sqa/github_backup.md`, so this should be found by all apps if they use the framework default.